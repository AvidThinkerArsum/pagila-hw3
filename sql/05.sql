/*
 * You love the acting in the movie 'AMERICAN CIRCUS' and want to watch other movies with the same actors.
 *
 * Write a SQL query that lists the title of all movies that share at least 1 actor with 'AMERICAN CIRCUS'.
 *
 * HINT:
 * This can be solved with a self join on the film_actor table.
 */
