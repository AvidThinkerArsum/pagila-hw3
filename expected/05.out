         title         
-----------------------
 ALABAMA DEVIL
 ALASKA PHANTOM
 ALIEN CENTER
 AMERICAN CIRCUS
 AMERICAN CIRCUS
 AMERICAN CIRCUS
 AMERICAN CIRCUS
 AMERICAN CIRCUS
 ANTITRUST TOMATOES
 ARABIA DOGMA
 ARABIA DOGMA
 ARTIST COLDBLOODED
 ATLANTIS CAUSE
 BADMAN DAWN
 BEACH HEARTBREAKERS
 BEETHOVEN EXORCIST
 BONNIE HOLOCAUST
 BOOGIE AMELIE
 BOONDOCK BALLROOM
 BOULEVARD MOB
 BRANNIGAN SUNRISE
 BREAKING HOME
 BUCKET BROTHERHOOD
 CANDIDATE PERDITION
 CHILL LUCK
 CHOCOLAT HARRY
 CITIZEN SHREK
 CLONES PINOCCHIO
 COMMANDMENTS EXPRESS
 CONEHEADS SMOOCHY
 CONFESSIONS MAGUIRE
 CONQUERER NUTS
 CRAFT OUTFIELD
 CROSSROADS CASUALTIES
 CROW GREASE
 CUPBOARD SINNERS
 CYCLONE FAMILY
 DALMATIONS SWEDEN
 DATE SPEED
 DESTINATION JERK
 DESTINY SATURDAY
 DOGMA FAMILY
 DONNIE ALLEY
 DOUBLE WRATH
 DROP WATERFRONT
 DURHAM PANKY
 EMPIRE MALKOVICH
 ENDING CROWDS
 ESCAPE METROPOLIS
 FICTION CHRISTMAS
 FINDING ANACONDA
 FIREBALL PHILADELPHIA
 FOOL MOCKINGBIRD
 FORREST SONS
 FREAKY POCUS
 FRONTIER CABIN
 GABLES METROPOLIS
 GAMES BOWFINGER
 GOODFELLAS SALUTE
 GORGEOUS BINGO
 GRACELAND DYNAMITE
 GREASE YOUTH
 GRINCH MASSAGE
 GUYS FALCON
 HALL CASSIDY
 HAMLET WISDOM
 HATE HANDICAP
 HEAVENLY GUN
 HEAVYWEIGHTS BEAST
 HOBBIT ALIEN
 HOCUS FRIDA
 HOLES BRANNIGAN
 IMPACT ALADDIN
 INVASION CYCLONE
 IRON MOON
 LANGUAGE COWBOY
 LIES TREATMENT
 LONELY ELEPHANT
 LORD ARIZONA
 LOVERBOY ATTACKS
 MADISON TRAP
 MADNESS ATTACKS
 MANNEQUIN WORST
 MASK PEACH
 MASSAGE IMAGE
 MASSAGE IMAGE
 MEET CHOCOLATE
 MERMAID INSECTS
 MISSION ZOOLANDER
 MONSTER SPARTACUS
 MUMMY CREATURES
 MUMMY CREATURES
 NAME DETECTIVE
 OKLAHOMA JUMANJI
 OLEANDER CLUE
 OSCAR GOLD
 OZ LIAISONS
 OZ LIAISONS
 PITTSBURGH HUNCHBACK
 PLATOON INSTINCT
 PLUTO OLEANDER
 POLISH BROOKLYN
 POLISH BROOKLYN
 POTTER CONNECTICUT
 PRIMARY GLASS
 RANGE MOONWALKER
 RINGS HEARTBREAKERS
 ROCKETEER MOTHER
 RUNAWAY TENENBAUMS
 RUN PACIFIC
 SABRINA MIDNIGHT
 SAINTS BRIDE
 SATURDAY LAMBS
 SHAKESPEARE SADDLE
 SHEPHERD MIDSUMMER
 SHIP WONDERLAND
 SIEGE MADRE
 SIEGE MADRE
 SKY MIRACLE
 SORORITY QUEEN
 SPIRITED CASUALTIES
 SPY MILE
 SQUAD FISH
 SUBMARINE BED
 SUBMARINE BED
 SUICIDES SILENCE
 SUNDANCE INVASION
 SUPERFLY TRIP
 SWEDEN SHINING
 TEXAS WATCH
 TITANS JERK
 TOWN ARK
 TRAIN BUNCH
 TRAMP OTHERS
 TRAMP OTHERS
 TREATMENT JEKYLL
 TROUBLE DATE
 TRUMAN CRAZY
 VERTIGO NORTHWEST
 WAKE JAWS
 WORLD LEATHERNECKS
 WRONG BEHAVIOR
 WYOMING STORM
(143 rows)

